docs/citation_style.jl

```diff
@@ -0,0 +1,38 @@
+#
+# This file is included by docs/make_work.jl to define the custom citation style `oscar_style`.
```

Comment:
@lgoettgens I think you wrote this file -- I copied it from Oscar.jl, but there is no mention of you. I'd be happy to add a copyright notice or whatever, let me know if you have a concrete suggestion. Or we leave it as-is, whichever you prefer.

(Both packages are under the same license, so that bit at least should be fine)

Reply:
Hmm, what about something like the following?
```
Copyright (c) 2023 Lars Göttgens, The OSCAR Development Team
This file was copied from https://github.com/oscar-system/Oscar.jl version 1.1.0
```
I think that referencing the Oscar repository is the most important thing here.

But then we should do something similar for the other file that you copied: `.bibtoolrsc`. Either with 2021-2023 and both our names (according to its history https://github.com/oscar-system/Oscar.jl/commits/master/.bibtoolrsc), or I would be fine with just The OSCAR Development Team instead of individual names. And again add a reference to the Oscar repo, in the same style as for the other file.

Thanks a lot for thinking about such things!

Reply:
Resolved.

Though I personally really don't want to claim copyright on a simple config file like `.bibtoolrsc`.

Reply:
Thanks :)